tasks/push-snapshot/push-snapshot.yaml

```diff
@@ -99,17 +94,22 @@ spec:
           repository=$(jq -r '.repository' <<< $component)
           imageTags=$(jq '.tags' <<< $component)
 
+          AUTH_FILE=$(mktemp)
+          select-oci-auth "${containerImage}" > "$AUTH_FILE"
+
           # Just read the first from the list of architectures
           read -r arch_json <<< $(get-image-architectures "${containerImage}")
+          os=$(echo "${arch_json}" | jq -r .platform.os)
           arch=$(echo "${arch_json}" | jq -r .platform.architecture)
           name=$(jq -r '.name' <<< $component)
           git_sha=$(jq -r '.source.git.revision' <<< $component) # this sets the value to "null" if it doesn't exist
+          media_type=$(skopeo inspect --raw "docker://${containerImage}" | jq -r .mediaType)
```

Comment:
Wait, but you cannot do this, no? I thought `skopeo` won't work on non-image oci images, hence the move to `oras`. Or am I missing something?

Reply:
skopeo inspect with **--raw** is implemented for non-image OCI artifacts.

```
> skopeo inspect --raw docker://quay.io/rbean/testing:super-cool-content | jq  .mediaType
"application/vnd.oci.image.manifest.v1+json"

```

and 

```
> skopeo inspect --raw docker://quay.io/rbean/testing:super-cool-content | jq  .layers
[
  {
    "mediaType": "text/plain",
    "digest": "sha256:37980c33951de6b0e450c3701b219bfeee930544705f637cd1158b63827bb390",
    "size": 13,
    "annotations": {
      "org.opencontainers.image.title": "foo.txt"
    }
  }
]

```